signer/src/bitcoin/utxo.rs

```diff
@@ -80,7 +80,7 @@ const BASE_WITHDRAWAL_TX_VSIZE: f64 = MAX_BASE_TX_VSIZE as f64;
 
 /// This constant represents the maximum virtual size (in vBytes) of a BTC
 /// transaction excluding withdrawals outputs and deposit inputs.
-pub const MAX_BASE_TX_VSIZE: u64 = 164;
+pub const MAX_BASE_TX_VSIZE: u64 = 137;
```

Comment:
`BASE_WITHDRAWAL_TX_VSIZE` is (still?) equal to this one, is it right?

Reply:
Yeah, it's not obvious. The `BASE_WITHDRAWAL_TX_VSIZE` is computed by constructing a sweep with one withdrawal, popping the withdrawal out of the transaction, and then looking at the `vsize`. This ends up encoding the withdrawal ID onto the `OP_RETURN`, so it accounts for all of the baggage that way. Also, the withdrawal ID is set to `u64::MAX` to make sure that the base size is as large as can be.

Which reminds me. We can update the logic of the deposits to follow this same pattern, which would make things symmetric and easier to understand. But maybe I'll do that later. Also, we can ship without updating this since we're reducing the value. This change makes it so that more withdrawals aren't wrongfully excluded.